jdbc/basics/src/test/java/example/springdata/jdbc/basics/simpleentity/SimpleEntityTests.java

```diff
@@ -61,4 +61,13 @@ public void exerciseRepositoryForSimpleEntity() {
 		repository.delete(cars);
 		Output.list(repository.findAll(), "`Cars` is gone.");
 	}
+
+	@Test
+	public void directInsert(){
```

Comment:
Shouldn't we rather point out the simple-most approach which would be direct `JdbcAggregateOperations` usage?

Reply:
I'm not sure.
As the intense discussion about DATAJDBC-282 shows many people want the insert method in their repositories. 
With the custom method, they get just that. 

Also, I guess most Spring Data JDBC users come from Spring Data JPA and don't even know about templates. 

I'd rather leave it as it is.